Cargo.toml

```diff
@@ -1,7 +1,6 @@
 [workspace]
 members = [
     "snapshot-parser",
-    "snapshot-parser-types",
```

Comment:
I was unsure how this change influences how the bonds work (and if snapshot manager parsing works with the new version of snapshot). What I tested, both work. I would only suggest adding a bit of comment on that.
https://github.com/marinade-finance/solana-snapshot-parser/pull/10